src/dos/dos_programs.cpp

```diff
@@ -295,23 +296,34 @@ class MOUNT : public Program {
 					temp_line = lastconfigdir + CROSS_FILESPLIT + temp_line;
 				}
 			}
+
+#if defined(WIN32)
+			/* Removing trailing backslash if not root dir so stat
+			 * will succeed */
+			if (temp_line.size() > 3 &&
+			    temp_line[temp_line.size() - 1] == '\\')
+				temp_line.erase(temp_line.size() - 1, 1);
+#endif
+
+			const std::string real_path = to_native_path(temp_line);
+			if (real_path.empty()) {
+				LOG_MSG("MOUNT: Path '%s' not found", temp_line.c_str());
```

Comment:
Suggest `"Case-insensitive path '%s' not found"`, because the search for a match was broader than the exact string held in `temp_line`.

Reply:
I think when it's not found then distinction between case sensitive and insensitive doesn't really matter ;) Keeping cleaner log.